docs/statistics.md

```diff
@@ -0,0 +1,561 @@
+The current tolerance value is 1, which equals to the gap. No loss is reported. \
+Next packet with sequence number 10 has a higher sequence discontinuity equal to 2.
+Missing packets with sequence numbers 8 and 9 will be reported lost with the next received packet (reorder distance 1).
+The next received packet has sequence number 8. Reorder tolerance value is increased to 2.
+The packet with sequence number 9 is reported lost.
+
+## pktRcvAvgBelatedTime
+
+Accumulated difference between the current time and the time-to-play of a packet 
+that is received late.
+
+## pktRcvBelated
+
+The number of packets received but IGNORED due to having arrived too late.
+
+Makes sense only if TSBPD and TLPKTDROP are enabled.
+
+An offset between sequence numbers of the newly arrived DATA packet and latest 
+acknowledged DATA packet is calculated.
+If the offset is negative, the packet is considered late, meaning that it was 
+either already acknowledged or dropped by TSBPD as too late to be delivered.
+
+Retransmitted packets can also be considered late.
+
+## pktSndDrop
+
+Same as `pktSndDropTotal`, but on the interval.
+
+## pktRcvDrop
+
+Same as `pktRcvDropTotal`, but on the interval.
+
+## pktRcvUndecrypt
+
+Same as `pktRcvUndecryptTotal`, but on the interval.
+
+## byteSent
+
+Same as `byteSentTotal`, but on the interval.
+
+## byteRecv
+
+Same as `byteRecvTotal`, but on the interval.
+
+## byteRcvLoss
+
+Same as `byteRcvLossTotal`, but on the interval.
+
+## byteRetrans
+
+Same as `byteRetransTotal`, but on the interval.
+
+## byteSndDrop
+
+Same as `byteSndDropTotal`, but on the interval.
+
+## byteRcvDrop
+
+Same as `byteRcvDropTotal`, but on the interval.
+
+## byteRcvUndecrypt
+
+Same as `byteRcvUndecryptTotal`, but on the interval.
+
+# Instant measurements
+
+The measurements effective at the time retrieved.
+
+## usPktSndPeriod
+
+Current minimum time interval between which consecutive packets are sent, in 
+microseconds. Sender only.
+
+Note that several sockets sharing one outgoing port use the same sending queue.
+They may have different pacing of the outgoing packets, but all the packets will
+be placed in the same sending queue, that may affect actual sending timing.
+
+`usPktSndPeriod` is the minimum time (sending period) that must be kept
+between two sent consecutively sent packets over the link used by SRT socket.
+It is not the EXACT time interval between
+two consecutive packets are sent. In the case where the time spent by the 
+application between two consecutive sendings exceeds `usPktSndPeriod`, the next 
+packet will be sent faster or even immediately to preserve the average sending rate.
+
+**Note**. Except for probing packets.
+
+## pktFlowWindow
+
+The maximum number of packets that can be "in flight". Sender only.
+See also [pktFlightSize](#pktFlightSize).
+
+The value represents the amount of free space
+on the peer receiver, stating that this socket represents the sender. When this
```

Comment:
This first sentence is not clear to me.

Reply:
Tryed to rephrase.
The meaning is that this value is determined on the sender side, and it is an estimation of the amount of free space in the buffer of the receiver peer. The actual value is periodically reported back by the receiver in ACK packets.